linux/packages/ubuntu-20.04/debian/rules

```diff
@@ -14,14 +14,16 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 	-DCMAKE_BUILD_TYPE=Release \
 	-Daudacity_conan_allow_prebuilt_binaries=no \
-   -D audacity_conan_force_build_dependencies=yes \
+    -Daudacity_conan_force_build_dependencies=yes \
```

Comment:
Could we replace these preceding spaces with tabs so that they are consistent with the other -D defines?  Or replace all the other with same amount of spaces.